### DISCUSSION TITLE
v1.7.1 - Fix broken subtitles and fullscreen issue

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#272] [General]
This is a hotfix that fixes some important issues. Changes:

 - Fix audio and subtitle selection for 10.8.0. (#199)
 - Stop forcing fullscreen on Windows when visibility changes. (#94)
 - Update jellyfin-web client to 10.8.1.

If you have issues with media playback due to hardware decoding (green/purple or artifacts), you can [disable it using these instructions](https://github.com/jellyfin/jellyfin-media-player/wiki/Disabling-Hardware-Decoding).

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.7.1'>v1.7.1 - Fix broken subtitles and fullscreen issue</a>.</em>

**Comment by jasoncmuxlow** [comment]
The 1.7.1 macOS DMG seems to contain 1.7.0. 

Steps to reproduce:
* Open Jellyfin 1.7.0
* Click "Download" button in popup modal re: 1.7.1
* Click the 1.7.1 macOS link on the GitHub page
* Download JellyfinMediaPlayer-1.7.1-macos.dmg
* Open JellyfinMediaPlayer-1.7.1-macos.dmg
* Right click on Jellyfin Media Player app in DMG > Get Info
* > Version 1.7.0
* > Modified Jun 26, 2022

Is that right? 

PS: Thanks to all involved for all the hard work on this app.

**Reply by anthonylavado** [reply]
Hi Jason, thanks for bringing this up. There was a caching issue when I uploaded the signed build. Can you download it again and let me know if it's good?

**Reply by jasoncmuxlow** [reply]
Perfect. Thanks!

**Comment by Mattia98** [comment]
bookworm package does not install on Debian bookworm, since the dependency `libmpv1` does not exist anymore in bookworm, it has been replaced with `libmpv2`

https://packages.debian.org/bullseye/libmpv1
https://packages.debian.org/bookworm/libmpv2

**Reply by thornbill** [reply]
I would suggest creating an issue instead of commenting on a discussion for a specific release.
